tmc-prototype/CentralNode/CentralNode/CentralNode.py

```diff
@@ -251,99 +174,98 @@ def push_event(self, evt):
-            print "Error message is: \n", e
+
+        except Exception as except_occured:
+            print CONST.ERR_INIT_PROP_ATTR_CN
+            self._read_activity_message = CONST.ERR_INIT_PROP_ATTR_CN
+            self.devlogmsg(CONST.ERR_INIT_PROP_ATTR_CN, 2)
+            self._read_activity_message = CONST.STR_ERR_MSG + str(except_occured)
+            print CONST.STR_ERR_MSG, except_occured
 
         #  Get Dish Leaf Node devices List
-        self.db = PyTango.Database()
+        self.tango_db = tango.Database()
         try:
-            self.dev_DbDatum = self.db.get_device_exported("ska_mid/tm_leaf_node/d000*")
-            self._dish_leaf_node_devices.extend(self.dev_DbDatum.value_string)
+            self.dev_dbdatum = self.tango_db.get_device_exported(CONST.GET_DEVICE_LIST_TANGO_DB)
+            self._dish_leaf_node_devices.extend(self.dev_dbdatum.value_string)
             print self._dish_leaf_node_devices
 
-        except Exception as e:
-            print "Unexpected error in reading exported Dish Leaf Node device names from database \n", e
-            self._read_activity_message = "Unexpected error in reading exported Dish Leaf Node device names from database \n" + str(e)
-            self.devlogmsg(
-                "Unexpected error in reading exported Dish Leaf Node device names from database \n", 2)
+        except Exception as except_occured:
+            print CONST.ERR_IN_READ_DISH_LN_DEVS, except_occured
+            self._read_activity_message = CONST.ERR_IN_READ_DISH_LN_DEVS + str(except_occured)
+            self.devlogmsg(CONST.ERR_IN_READ_DISH_LN_DEVS, 2)
 
         # Create proxies of Dish Leaf Node devices
 
-        for name in range (0,len(self._dish_leaf_node_devices)):
+        for name in range(0, len(self._dish_leaf_node_devices)):
             try:
                 self._leaf_device_proxy.append(DeviceProxy(self._dish_leaf_node_devices[name]))
 
-            except Exception as e:
-                print "Unexpected error in creating proxy of the device ", self._dish_leaf_node_devices[name]
-                self._read_activity_message = "Unexpected error in creating proxy of the device " +  str(self._dish_leaf_node_devices[name])
-                print "Error message is: \n", e
-                self._read_activity_message = "Error message is: \n" + str(e)
-                self.devlogmsg(
-                    "Unexpected error in creating proxy of the device ", 2)
+            except Exception as except_occured:
+                print CONST.ERR_IN_CREATE_PROXY, self._dish_leaf_node_devices[name]
+                self._read_activity_message = CONST.ERR_IN_CREATE_PROXY \
+                                              + str(self._dish_leaf_node_devices[name])
+                print CONST.STR_ERR_MSG, except_occured
+                self._read_activity_message = CONST.STR_ERR_MSG + str(except_occured)
+                self.devlogmsg(CONST.ERR_IN_CREATE_PROXY, 2)
         print self._leaf_device_proxy
```

Comment:
Can remove the print statements used for testing.

Reply:
Removed unnecessary print statements. Print statements at some places are helpful to indicate errors and exceptions. Therefore not removing all of them.